packages/in_app_purchase/in_app_purchase_android/android/src/main/java/io/flutter/plugins/inapppurchase/MethodCallHandlerImpl.java

```diff
@@ -131,6 +130,7 @@ void onDetachedFromActivity() {
   }
 
   @Override
+  @SuppressWarnings(value = "deprecation")
```

Comment:
Generally we extract deprecated code paths to a minimal helper method that we can mark as deprecated, rather than making a whole large method, since the latter will hide future issues.

This also seems like something we should have an issue and TODO for migrating to the new API.